pkg/targetallocator/deployment_test.go

```diff
@@ -17,13 +17,28 @@ package targetallocator
 import (
 	"testing"
 
+	v1 "k8s.io/api/core/v1"
```

Comment:
I think this import should be in the group below

Reply:
Done